src/ResourceManager/Reservations/Commands.Reservations/help/Get-AzureRmReservation.md

```diff
@@ -0,0 +1,124 @@
+---
+external help file: Microsoft.Azure.Commands.Reservations.dll-Help.xml
+Module Name: AzureRM.Reservations
+online version: 
```

Comment:
Please add in the online version string as in here: https://raw.githubusercontent.com/Azure/azure-powershell/preview/src/ResourceManager/Compute/Commands.Compute/help/Add-AzureRMVhd.md

Reply:
online version added for each md files